src/components/Books/Book/Book.jsx

```diff
@@ -1,18 +1,28 @@
+/* eslint-disable no-unused-vars */
 import React from 'react';
-
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 
+// import { removeBook } from '../../../redux/books/books';
+
 const books = [
-  { id: 'a1', name: 'book1' },
+  { id: 1, name: 'book1' },
   { id: 2, name: 'book2' },
 ];
 
-const Book = () => books.map((book) => (
-  <li key={book.id}>
-    {book.name}
-    <button type="button">remove</button>
-  </li>
-));
+const Book = () => {
+  const dispatch = useDispatch();
+  const removeBookFromStore = () => {
+    // dispatch(removeBook('newBook'));
+  };
+
```

Comment:
- [ ] Please work on the `remove` feature I can see that you started from these codes but removeBook reducer is commented work on that and always remember to remove commented codes